### PR TITLE
Added Computer Vision sample quickstarts

### PR DESCRIPTION
**Pull request by harishkrishnav**
## Purpose
Added quickstarts

## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
Adding samples

```
[ ] Bugfix
[ ] Feature
[x] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: dotnet/ComputerVision/BatchReadText/Program.cs [hunk 1]
```diff
@@ -5,7 +5,7 @@
 using System.Net.Http.Headers;
 using System.Threading.Tasks;
 
-namespace Microsoft.Azure.CognitiveServices.Samples.ComputerVision.ExtractText
+namespace Microsoft.Azure.CognitiveServices.Samples.ComputerVision.BatchReadText
```
Comment (CarlosP-MS) [inline dotnet/ComputerVision/BatchReadText/Program.cs]: BatchReadFile for consistency with SDK version

File: dotnet/ComputerVision/BatchReadText/Program.cs [hunk 2]
```diff
@@ -32,12 +32,12 @@ public static async Task RunAsync(string endpoint, string key)
             string imageFilePath = @"Images\handwritten_text.jpg"; // See this repo's readme.md for info on how to get these images. Alternatively, you can just set the path to any appropriate image on your machine.
             string remoteImageUrl = "https://github.com/Azure-Samples/cognitive-services-sample-data-files/raw/master/ComputerVision/Images/printed_text.jpg";
 
-            await ExtractTextFromStreamAsync(imageFilePath, endpoint, key, "Handwritten");  //the last parameter is whether the text that has to be extracted is printed or handwritten 
-            await ExtractTextFromUrlAsync(remoteImageUrl, endpoint, key, "Printed"); //This textRecognitionMode can only be either Handwritten or Printed
+            await BatchReadTextFromStreamAsync(imageFilePath, endpoint, key, "Handwritten");  //the last parameter is whether the text that has to be extracted is printed or handwritten
```
Comment (CarlosP-MS) [inline dotnet/ComputerVision/BatchReadText/Program.cs]: Remove the mode parameter and the corresponding http requests, just like in the SDK version.

File: dotnet/ComputerVision/RecognizeDomainSpecificContent/Program.cs [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+                Console.WriteLine("\nInvalid file path");
+                return;
+            }
+            try
+            {
+                HttpClient client = new HttpClient();
+
+                // Request headers.
+                client.DefaultRequestHeaders.Add("Ocp-Apim-Subscription-Key", subscriptionKey);
+                string uri = endpoint + @"/vision/v2.0/models/"+specificDomain+ @"/analyze";
```
Comment (CarlosP-MS) [inline dotnet/ComputerVision/RecognizeDomainSpecificContent/Program.cs]: Lines like this look better as 
string uri = $"{endpoint}/vision/v2.0/models/{specificDomain}/analyze"

File: dotnet/ComputerVision/RecognizeText/Program.cs [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+    public class RecognizeTextSample
+    {
+        public static async Task RunAsync(string endpoint, string key)
+        {
+            Console.WriteLine("Recognizing text from the images:");
+
+            string imageFilePath = @"Images\handwritten_text.jpg"; // See this repo's readme.md for info on how to get these images. Alternatively, you can just set the path to any appropriate image on your machine.
+            string remoteImageUrl = "https://github.com/Azure-Samples/cognitive-services-sample-data-files/raw/master/ComputerVision/Images/printed_text.jpg";
+
+            await RecognizeTextFromStreamAsync(imageFilePath, endpoint, key, "Handwritten");  //the last parameter is whether the text that has to be extracted is printed or handwritten
```
Comment (CarlosP-MS) [inline dotnet/ComputerVision/RecognizeText/Program.cs]: Same as the SDK case, we no longer need the mode parameter

File: dotnet/ComputerVision/readme.md [hunk 5]
```diff
@@ -17,8 +17,13 @@ The projects in this code are quickstarts that show how to use Computer Vision R
 |-------------|-------------|
 | AnalyzeImage       | Project that analyzes image on specified visual parameters |
 | DetectObjects | Project that detects objects in an image |
-| ExtractText | Project that looks for printed or handwritten text in an image|
+| BatchReadText | Project that looks for printed or handwritten text in an image|
```
Comment (CarlosP-MS) [inline dotnet/ComputerVision/readme.md]: Update to match SDK case

File: dotnet/ComputerVision/RecognizeText/Program.cs (hunk 4 above)
Comment (harishkrishnav) [reply dotnet/ComputerVision/RecognizeText/Program.cs]: https://westus.dev.cognitive.microsoft.com/docs/services/5adf991815e1060e6355ad44/operations/587f2c6a154055056008f200
It is required, like in the SDK case

File: dotnet/ComputerVision/RecognizeDomainSpecificContent/Program.cs [hunk 6]
```diff
@@ -48,7 +48,8 @@ static async Task RecognizeDomainSpecificContentFromStreamAsync(string imageFile
 
                 // Request headers.
                 client.DefaultRequestHeaders.Add("Ocp-Apim-Subscription-Key", subscriptionKey);
-                string uri = endpoint + @"/vision/v2.0/models/{specificDomain}/analyze";
+                string uri = endpoint + $@"/vision/v2.0/models/{specificDomain}/analyze";
```
Comment (CarlosP-MS) [inline dotnet/ComputerVision/RecognizeDomainSpecificContent/Program.cs]: Couple of things:
- You don't need the @ sign in that particular string.
- We should put the {endpoint} variable inside the string as well, so it is all consistent. 

Let's do that for the other places where you build these URIs (not just in this file, but the other Rest samples too).

**Pull request merged**